src/scss/components/search/_index.scss

```diff
@@ -61,6 +61,7 @@ hx-search-control {
   > input[type="search"]:focus ~ hx-search,
   > button.hxClear:focus ~ hx-search {
     border-color: $cyan-700;
```

Comment:
Please use updated  `v2.0` colors to replace `$cyan-XXX`.

Reply:
Changes have been done, Please review